airflow/models/baseoperator.py

```diff
@@ -633,6 +670,51 @@ def deps(self) -> Set[BaseTIDep]:
             NotPreviouslySkippedDep(),
         }
 
+    def lock_for_execution(self) -> None:
+        """Sets _lock_for_execution to True"""
```

Comment:
This description is somewhat self-referrential - I could have guess this is what it did based on the name of the method.

Either say what this setting actually does, or remove the doc comment please -- having this doc doesn't add any value